src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -121,7 +123,7 @@ ZoneWindow::ZoneWindow(
     if (m_window)
     {
         MakeWindowTransparent(m_window.get());
-        if (flashZones)
+        if (flashZones && !IsFullscreen(GetForegroundWindow()))
```

Comment:
The caller of IsFullscreen, has already called GetMonitorInfo and retrieved the monitor rect, can we pass it to IsFullscreen? Or remove IsFullscreen and do the rect check inline, inside `if (flashZones)`.

Reply:
We can use monitor info obtained previously in this method, but I would keep `IsFullscreen` function, so it's more clear what we are trying to achieve.